client/extensions/archive-search/src/extension.ts

```diff
@@ -0,0 +1,24 @@
+import {
+  IExtension,
+  IExtensionActivationResult,
+  ISearchPanelWidgetProps,
+} from "superdesk-api";
+import { widgetFactory } from "./widget";
+
+const extension: IExtension = {
+  activate: () => {
+    const result: IExtensionActivationResult = {
+      contributions: {
+        searchPanelWidgets: [
+          widgetFactory() as React.ComponentType<
```

Comment:
we did improvements in our extensions framework so factories are no longer needed. You can use components directly:

`searchPanelWidgets: [SearchPanelWidget]`